<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-C4NNM2J" failures="2" tests="2" name="com.testcases.Tests_LoginModule" time="16.215" errors="0" timestamp="2023-03-07T17:53:47 GMT" skipped="0">
  <testcase classname="com.testcases.Tests_LoginModule" name="runScenario" time="6.592">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.steps.pages.LoginPage.userCredentialsCheck(LoginPage.java:51)
at ✽.The login page should display error with fail status(file:///C:/Users/17kri/eclipse-workspace/QAAutomationTask/src/main/java/features/Login.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.testcases.Tests_LoginModule" name="runScenario" time="9.623">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.steps.pages.ForgotPassword.forgotPAssword(ForgotPassword.java:35)
at ✽.The forgot page should display error with fail status(file:///C:/Users/17kri/eclipse-workspace/QAAutomationTask/src/main/java/features/Login.feature:15)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- com.testcases.Tests_LoginModule -->
